core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -255,6 +255,7 @@ oppia.directive('conversationSkin', [
         'siteAnalyticsService', 'ExplorationPlayerStateService',
         'TWO_CARD_THRESHOLD_PX', 'CONTENT_FOCUS_LABEL_PREFIX', 'alertsService',
         'CONTINUE_BUTTON_FOCUS_LABEL', 'EVENT_ACTIVE_CARD_CHANGED',
+        'fatigueDetectionService',
```

Comment:
Done. Spent some time deliberating on this because I couldn't decide what was deciding camelCase vs. UpperCamelCase for services